src/ansys/fluent/visualization/matplotlib/matplot_windows_manager.py

```diff
@@ -280,15 +280,15 @@ def save_graphic(
         window_id: str,
         format: str,
     ) -> None:
-        """Save graphics.
+        """Save a graphic.
 
         Parameters
         ----------
         window_id : str
-            Window id for which graphic should be saved.
+            Window ID for the graphic to save.
         format : str
-            Graphic format. Supported formats are eps, jpeg, jpg,
-            pdf, pgf, png, ps, raw, rgba, svg, svgz, tif and tiff.
+            Graphic file format. Supported formats are EPS, JPEG, JPG,
```

Comment:
When an extension is not used as part of a specific file name, it is to appear in all capital letters.